docs/developers.md

```diff
@@ -149,8 +149,5 @@ Once it is running without bugs on your local fork, request a *Pull request* by:
 * On the right-hand side select your fork, for example: *head repository: BFH-ktt1/logisim-evolution*
 * On the right-hand side select your branch, for example: *base: bugfixes*
 * On the left-hand side select the main branch *base: main* (**Important:** All pull requests **MUST**
-  be on the [branch `main`](https://github.com/logisim-evolution/logisim-evolution/tree/main) as
-  the [branch `master`](https://github.com/logisim-evolution/logisim-evolution/tree/master) only
-  holds the code of the latest stable release, and we do not allow any external contributions to that
-  particular branch.
+  be made on the branch [`main`](https://github.com/logisim-evolution/logisim-evolution/tree/main).
```

Comment:
Maybe: "be made against [that branch](https://github.com/logisim-evolution/logisim-evolution/tree/main)."

Reply:
Done.